@@
@@ Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
@@ This program is free software: you can redistribute it and/or modify
@@ it under the terms of the GNU Affero General Public License as published
@@ by the Free Software Foundation, either version 3 of the License, or
@@ (at your option) any later version.
@@ This program is distributed in the hope that it will be useful,
@@ but WITHOUT ANY WARRANTY; without even the implied warranty of
@@ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
@@ GNU Affero General Public License for more details.
@@ You should have received a copy of the GNU Affero General Public License
@@ along with this program. If not, see <https://www.gnu.org/licenses/>.
@@ For any questions about this software or licensing,
@@ please email opensource@seagate.com or cortx-questions@seagate.com.
@@


diff --git a/lib/trace.c b/lib/trace.c
index f453deb..fbd97c3 100644
--- a/lib/trace.c
+++ b/lib/trace.c
@@ -214,6 +214,8 @@ M0_INTERNAL void m0_trace_allot(const struct m0_trace_descr *td,
 	struct m0_trace_buf_header *tbh = m0_logbuf_header;
 	register unsigned long      sp asm("sp"); /* stack pointer */

+	return;
+
 #ifdef ENABLE_RESTRICTED_TRACE_MODE
 	/* discard records with verbosity level higher than allowed */
 	if (td->td_level > M0_TRACE_HIGHEST_ALLOWED_LEVEL)
